domains/files.allancoding.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "allancoding",
+        "email": "allancoding.dev@gmail.com"
+    },
+    "record": {
+        "CNAME": "allancoding.github.io"
```

Comment:
404 CNAME record.

Reply:
It is here: https://allancoding.github.io/files.allancoding.is-a.dev/